### ISSUE TITLE
Spanish translation. (Es-es)

### ISSUE THREAD

**Issue by caralu74** [tuxedocomputers/tuxedo-control-center#19]
Hello!
I can help to localize this tool to Spanish.
Do you have .po files? 
May I help?

**Comment by caralu74** [comment]
Waiting...

**Comment by nickma82** [comment]
Status on this plz? @caralu74 

Did you see the lang assets there: https://github.com/tuxedocomputers/tuxedo-control-center/tree/master/src/ng-app/assets/locale
Best, Nick

**Comment by caralu74** [comment]
Yes, I saw them (after a lot of searching, cuz I thought Spanish wasn't a good option) but I could no open the XLIFF´s the editor (poeditor) keep saying they are malformed. Could you please recommend some xliff editor?
I thought Spanish wasn't a good option

‐‐‐‐‐‐‐ Original Message ‐‐‐‐‐‐‐

**Comment by tuxedoxt** [comment]
Hello, poedit should work fine. Make sure you take the latest version from their website, some repos seem to contain older versions that does not support xliff yet.

**Comment by Gianvacca** [comment]
Hi, I'm using poetit 2.4.1 and if I try to open the file 'lang.xlf' I get the error message 'Error loading file "home/user/Documents/lang.xlf": Error parsing element attribute.'
The same applies lang.en.xlf and lang.de.xlf.

**Comment by caralu74** [comment]
I was looking a little, so I think I will give a try to OmegaT, (Some tips about it)
I've just uninstalled poedit, even thought it works very well for other projects, but I have not enough space on my drive.

‐‐‐‐‐‐‐ Original Message ‐‐‐‐‐‐‐

**Comment by tuxedoxt** [comment]
> Hi, I'm using poetit 2.4.1 and if I try to open the file 'lang.xlf' I get the error message 'Error loading file "home/user/Documents/lang.xlf": Error parsing element attribute.'
> The same applies lang.en.xlf and lang.de.xlf.

Very weird. I just tried it with that exact version successfully.

Is it the latest version from the master branch? It seems your file is out of the repo structure, maybe something went wrong while downloading. Does it look like an xliff file?

**Comment by caralu74** [comment]
I will try again with Poeditor. Is there a way like md5, sha2 or similar to check the files?

Sent from [ProtonMail](https://protonmail.ch), encrypted email based in Switzerland.

Sent with [ProtonMail](https://protonmail.com) Secure Email.

‐‐‐‐‐‐‐ Original Message ‐‐‐‐‐‐‐

**Comment by tuxedoxt** [comment]
> Is there a way like md5, sha2 or similar to check the files?

One easy way to make sure you have the right version would be to just clone the repo locally with git.

**Comment by caralu74** [comment]
Thank you, for your answer!

**Comment by caralu74** [comment]
Neither Poeditor nor OmegaT nor XLIFF translator tool could open the .xlf files.

‐‐‐‐‐‐‐ Original Message ‐‐‐‐‐‐‐

**Comment by caralu74** [comment]
"CONSTANCY OVERCOMES WHAT BLISS CANNOT." Finally, I can open the file, so I will work on it.

**Comment by caralu74** [comment]
Übersetzung abgeschlossen!
Translation completed!
I hope it will be useful to more people.
